config/locales/en/pvb2-mailers.yml

```diff
@@ -106,6 +106,10 @@ en:
         Your visit can't take place as the prisoner has a restriction.
       child_protection_issues:
         Your visit can't take place as the prisoner has a restriction.
+      duplicate_visit:
+        You have already requested a visit with these details. You will have received
+        an email regarding your original visit request, containing link to check the
+        status of your request.
```

Comment:
@mloughran do you mean a link to the original request, or a link to the duplicate request?